tests/unit/fair_queue_test.cc

```diff
@@ -55,15 +55,15 @@ struct request {
 };
 
 class test_env {
-    fair_group _fg;
+    shared_throttle _fg;
```

Comment:
Minor: in the next patch we rename `_fgs` from io_queue to `_throttle` and then adjust io_queue's tests. I know that this particular test is related to fair_queue and not io_queue. I am just wondering if `_fg` here should also be adjusted.